src/planscape/forsys/spatial_clustering.R

```diff
@@ -0,0 +1,80 @@
+suppressMessages({
+    library(sf)
+    # TODO: switch?
+    library(tidyverse)
+})
+
+normalize_values <- function(x) {
+  (x - min(x)) / (max(x) - min(x))
+}
+
+cluster_cells_to_stands <- function(
```

Comment:
nit: spatial_clustering is a little general: can you please rename it to something more specific, such as kmeans clustering? Similarly, can you please rename the function cluster_cells_to_stands to something that includes "kmeans"?

As well, I'm guessing "spatial" alludes to the features used. Can you please add a comment somewhere listing the similarity feature used?

Reply:
Done.